src/TaskManager/Plug-ins/Argo/ArgoPlugin.cs

```diff
@@ -70,7 +74,7 @@ public sealed class ArgoPlugin : TaskPluginBase, IAsyncDisposable
             Initialize();
         }
 
-        private void Initialize()
+        private async void Initialize()
```

Comment:
this should return a task

Reply:
in fact Initialize is called from the constructer so  youll need to wrap your code in a task.run i think